From b94f55e35df9a868259720c82d5e5396e5989b93 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Wed, 14 Feb 2024 19:47:11 +0100
Subject: [PATCH] Allow to force a version

---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ea74d55..eacfc24 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -292,7 +292,10 @@ endif()
 
 string(REGEX MATCH ".*dev.*" DUCKDB_EXTENSION_FOLDER_IS_VERSION "${DUCKDB_VERSION}")
 
-if(DUCKDB_EXTENSION_FOLDER_IS_VERSION AND DEFINED GIT_COMMIT_HASH)
+if(DUCKDB_FORCE_VERSION)
+  set(DUCKDB_VERSION "${DUCKDB_FORCE_VERSION}")
+  set(DUCKDB_NORMALIZED_VERSION "${DUCKDB_FORCE_VERSION}")
+elseif(DUCKDB_EXTENSION_FOLDER_IS_VERSION AND DEFINED GIT_COMMIT_HASH)
   set(DUCKDB_NORMALIZED_VERSION "${GIT_COMMIT_HASH}")
 else()
   set(DUCKDB_NORMALIZED_VERSION "${DUCKDB_VERSION}")
-- 
2.43.0

