From 278813e7fc58df69bd8426cb389da19b621d4992 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Mon, 22 Jul 2024 11:09:30 +0200
Subject: [PATCH 2/2] Remove code signature before applying metadata to
 extension

---
 scripts/append_metadata.cmake | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/append_metadata.cmake b/scripts/append_metadata.cmake
index fc12587..0c0eade 100644
--- a/scripts/append_metadata.cmake
+++ b/scripts/append_metadata.cmake
@@ -15,6 +15,10 @@ set(META6 "" CACHE STRING "Metadata field")
 set(META7 "" CACHE STRING "Metadata field")
 set(META8 "" CACHE STRING "Metadata field")
 
+if (APPLE)
+    execute_process(COMMAND "/usr/bin/codesign" "--remove-signature" "${EXTENSION}")
+endif()
+
 # null.txt should contain exactly 1 byte of value \x00
 file(READ "${NULL_FILE}" EMPTY_BYTE)
 
