From 77e171228bc850599937aabf9487e8ba52ac141a Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Fri, 15 Dec 2023 20:18:15 +0100
Subject: [PATCH 1/6] Dynamically link the cli

---
 tools/shell/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/shell/CMakeLists.txt b/tools/shell/CMakeLists.txt
index 2cc7e13025..6881d834a2 100644
--- a/tools/shell/CMakeLists.txt
+++ b/tools/shell/CMakeLists.txt
@@ -30,7 +30,7 @@ set(SHELL_SOURCES

 add_executable(shell ${SHELL_SOURCES})

-target_link_libraries(shell duckdb_static ${DUCKDB_EXTRA_LINK_FLAGS})
+target_link_libraries(shell duckdb ${DUCKDB_EXTRA_LINK_FLAGS})
 link_threads(shell "")
 if(STATIC_LIBCPP)
   message("Statically linking CLI")
